README.md

```diff
@@ -20,7 +20,58 @@ spago install machines
 
 ## Quick start
 
-The quick start hasn't been written yet. Contributions are welcome!
+The examples here use `Identity` for simplicity. Usually, you would use
+a different `Monad`, such as `Effect`, `Aff`, `State`, etc.
+
+There are several ways to build machines. One way is to use `do` syntax,
+for example:
+
+```purescript
+machine1 :: MealyT Identity Unit String
+machine1 = do
+  number <- fromArray [10, 20, 30, 40, 50, 0, 60, 70]
+  scaled <-
+      if number == 0
+          then halt
+          else pure $ div number 2
+  pure $ show scaled
+```
+
+This will create a machine `machine1` which goes through the "inputs"
+from the array. It then checks and halts on any zero input, and otherwise
+scales the inputs (by dividing by 2). The result is then transformed intoa string.
+
+The resulting machine can be materialized via `toUnfoldable unit machine1 :: Array String`. The result is `["5","10","15","20","25"]`.
```

Comment:
We usually represent this with a pseudo-repl session:

---

The resulting machine can be materialized by transforming it into an array:

```purs
> toUnfoldable unit machine1 :: Array String
["5","10","15","20","25"]
```

---

though it's not necessary to do so. Just if you'd like!